halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AbstractAddAccountCommand.java

```diff
@@ -42,10 +43,17 @@
 
   @Parameter(
       variableArity = true,
-      names = "--required-group-membership",
-      description = AccountCommandProperties.REQUIRED_GROUP_MEMBERSHIP_DESCRIPTION
+      names = "--read-permissions",
```

Comment:
These flags may no longer be functional in the latest spinnaker release, but still work in older ones. I would not remove them here.

Reply:
Done

Reply:
@lwander 
Is there a deprecation process by which these flags should be removed at some time in the future?

Reply:
There is an annotation for flags that don't work _until_ a certain release -- we need to add/extend that annotation for flags that no longer work _after_ a certain release. That can be done in a separate PR.